alphabase/constants/spectral_library.py

```diff
@@ -0,0 +1,6 @@
+"""Constants for spectral library related functionality."""
+
+# Loss type mappings - these map loss type numeric codes to their string representations
+# Used in annotate.py and flat.py
+
+LOSS_NUMBER_TO_TYPE = {0: "", 17: "_NH3", 18: "_H2O", 98: "_modloss"}
```

Comment:
I think this can be removed as it's not needed by the new AB based implementation :).